sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsRequestOptions.java

```diff
@@ -11,7 +11,7 @@
 @Fluent
 public final class TextAnalyticsRequestOptions {
     private String modelVersion;
-    private boolean showStatistics;
+    private boolean isStatisticsShown;
```

Comment:
Consider changing to `IncludeStatistics`.
`isStatisticsShown` sounds like the service already returns them and we just want to show them, when in reality we are asking the service for those values so we can show them

Reply:
Java can't use `include` as prefix. https://azure.github.io/azure-sdk/java_design.html#model-classes
✅ DO use the JavaBean naming convention of get*, set*, and is*.

Reply:
I can change the private variable to `includeStatistics` but method name need to include is* as prefix. How is that sounds to you? So the method name is `isIncludeStatistics`.

https://azure.github.io/azure-sdk/java_design.html#model-classes

 ✅ DO use the JavaBean naming convention of get*, set*, and is*.